src/test/java/cz/it4i/fiji/datastore/startDataStoreServer.java

```diff
@@ -0,0 +1,19 @@
+package cz.it4i.fiji.datastore;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+@QuarkusTest()
+public class startDataStoreServer {
```

Comment:
Consider using UpperCase letter for class identifier.

Reply:
sorry, fixed (how could I missed that...)